unyt/unit_systems.py

```diff
@@ -298,7 +304,9 @@ def has_current_mks(self):
 
 
 #: The CGS unit system
-cgs_unit_system = UnitSystem("cgs", "cm", "g", "s", current_mks_unit=None)
+cgs_unit_system = UnitSystem(
+    "cgs", "cm", "g", "s", currency_unit="$", current_mks_unit=None
+)
```

Comment:
Since `currency_unit` defaults to `"$"` you don't need to edit the definitions of any of the built-in unit systems you've edited here or below